src/EditorFeatures/CSharpTest/Diagnostics/HideBase/HideBaseTests.cs

```diff
@@ -178,5 +178,123 @@ class App : Application
 {
     /* start */ public /* middle */ new readonly /* end */ int Test;
 }");
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveNew)]
+        public async Task TestRemoveNewFromProperty()
+        {
+            await TestInRegularAndScriptAsync(
+@"class App
+{
+    [|public static new App Current|] { get; set; }
```

Comment:
span seems weird.  would expect it to only be on `new` keyword.

Reply:
Could you please point me to the place where I can read about this syntax `[|. . .|]`? I just copied tests in this file for adding 'new' modifier.

Reply:
I think it represents what the text selection is in the text editor when Ctrl+. is pressed.

Reply:
Thanks @jnm2 , in this case I'd put it on the actual member name. @CyrusNajmabadi does it make sense for you?

Reply:
i would put it on the 'new' modifier since that's what' sbeing removed :)

Reply:
That's strange but in this case all tests are failing with `System.Exception: No action was offered when one was expected. Diagnostics from the compilation:` ("diagnostics from the compilation" looks empty). I've pushed it to a temp branch to not to trigger CI for known failed tests: https://github.com/vanashimko/roslyn/commit/976734b4308aa8c7fbc3f81a766b2e7710c7ae46. In case of putting this on member name, all tests pass. Can this be related not to the code fix itself, but to the way how this diagnostic applied?

Reply:
Yes, I think you have no choice but to make the span include the member name token unless you make changes in the compiler layer which I believe @CyrusNajmabadi would not recommend for this PR.

Reply:
> In case of putting this on member name


That's fine then :)

> Yes, I think you have no choice but to make the span include the member name token unless you make changes in the compiler layer which I believe @CyrusNajmabadi would not recommend for this PR.

Correct :)